particle-services/src/app_services.rs

```diff
@@ -826,6 +841,7 @@ impl ParticleAppServices {
             }
         });
 
+        let mut loaded_service_count = 0;
```

Comment:
suggest `created_service_count` or `created`. Because function is called `create_persisted_services`

```suggestion
        let mut created_service_count = 0;
```

Reply:
Fixed